app/model/ProgressStatuses.scala

```diff
@@ -121,6 +123,7 @@ object ProgressStatuses {
   case object ASSESSMENT_CENTRE_PASSED_NOTIFIED extends ProgressStatus(ApplicationStatus.ASSESSMENT_CENTRE_PASSED_NOTIFIED)
   case object ASSESSMENT_CENTRE_FAILED_NOTIFIED extends ProgressStatus(ApplicationStatus.ASSESSMENT_CENTRE_FAILED_NOTIFIED)
 
+
```

Comment:
You're fairly whitespace happy in this PR, i'm not holding it against you so far